common/src/fft/kiss_fft.c

```diff
@@ -255,7 +255,15 @@ void kf_work(
         int k;
 
         // execute the p different work units in different threads
-#       pragma omp parallel for
+#if defined _OPENMP && _OPENMP <= 201307 || (__GNUC__ >= 6 && __GNUC__ < 9)
```

Comment:
This expression is repeated in multiple files and looks extremely cryptic. What does it test, why? I'd suggest we give it a descriptive name (e.g. `OPENMP_SUPPORTS_XYZ`) and put it in `pcl_macros.h`.

Reply:
I added `OPENMP_LEGACY_CONST_DATA_SHARING_RULE` now (except in `kiss_fft.c` as this is a C file and can't include `pcl_macros.h`)

Reply:
Could you create a todo-issue because one of my branch's pcl_macros addresses this issue? That'll be resolved after a few merges though.

Reply:
Sorry for late response. Did you already have a PR for your changes related to  pcl_macros or an issue where I can attach this information?

Reply:
Yeah, if you rebase, you can now use `pcl_macros.h` as a C header

Reply:
#3545

Reply:
Perfect, thank you! I will adjust it.